src/__tests__/Search.test.js

```diff
@@ -0,0 +1,20 @@
+import React, { render, screen } from "@testing-library/react";
+import Search from "../components/Search";
+import setSearchResults from "../components/App";
```

Comment:
This shouldnt be imported, the validProps.setSearchResults should be passed into the render method instead